DOCS/man/options.rst

```diff
@@ -4087,6 +4087,17 @@ Input
 ``--input-cmdlist``
     Prints all commands that can be bound to keys.
 
+``--input-commands=<cmd1,cmd2,...>``
+    Define a list of commands for mpv to run. The syntax is the same as format
+    as ``input.conf`` but without the key binding argument at the beginning.
+
```

Comment:
Can we define the point in time when these are executed?
Seems to be once mpv enters idle mode.

Reply:
3 possibilities technically. If you pass `--idle` but with no actual file then it should happen almost immediately since otherwise it would do nothing (stuck in the idle loop). On the normal case, `mpv file.mkv --input-commands=something`, then it should execute after audio and/or video is ready and playback can truly start. And then finally if you change it at runtime it will happen immediately.

Reply:
Only now do I realize that you were obviously asking to document this more clearly. Done now.